### PR TITLE
JAMES-2233 Improve WebAdmin audit log

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by trantienduchn** [comment]
```
[2bd5e1bcd668f4194c6a5cb49744de8db97315a3] [ERROR] Errors: 
[2bd5e1bcd668f4194c6a5cb49744de8db97315a3] [ERROR]   ForwardSmtpTest.forwardingAnEmailShouldWork:79 » ConditionTimeout Assertion co...
```
test this please

**Comment by Arsnael** [comment]
```
[2bd5e1bcd668f4194c6a5cb49744de8db97315a3] [ERROR] Errors: 
[2bd5e1bcd668f4194c6a5cb49744de8db97315a3] [ERROR]   ForwardSmtpTest.forwardingAnEmailShouldWork:79 » ConditionTimeout Assertion co...
```
shouldn't be an issue anymore now...

test this please

File: server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/mdc/LoggingRequestFilter.java [hunk 1]
```diff
@@ -19,21 +19,34 @@
...
+import com.google.common.collect.ImmutableSet;
+
 import spark.Filter;
 import spark.Request;
 import spark.Response;
 
 public class LoggingRequestFilter implements Filter {
     private static final Logger LOGGER = LoggerFactory.getLogger(LoggingRequestFilter.class);
-    private static final String REQUEST_BODY = "request-body";
+    static final String REQUEST_BODY = "request-body";
```
Comment (remk) [inline server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/mdc/LoggingRequestFilter.java]: Why package visibility ?  i don't see it used elsewhere.

**Comment by chibenwa** [comment]
Just rebased

**Comment by chibenwa** [comment]
```
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607] [ERROR] Failures: 
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607] [ERROR]   GroupMappingTest.messageShouldRedirectToUserDoesNotHaveSameDomainWhenBelongingToGroup:182 
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607] Expecting:
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607]  <"AAAD OK FETCH completed.
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607] ">
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607] to contain:
[45f6f8e38cf6b9d80f592c03ed4b09a3a5aa3607]  <"any text"> 
```

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**